README.md

```diff
@@ -1,45 +1,8 @@
-# Ruby on Rails Learning Path
```

Comment:
You should not change the README file, this makes me think that you worked on the wrong base branch, anyway just be careful when you are branching cuz it might cause some problems/confusion when working on a client.